Develop client code to get bearer tokens with read and write scopes s…

Kravchook opened the pull request:
…eparately. Using RestSharp AuthenticatorBase develop custom authenticators that get token with “read” and “write” scope (check rest sharp documentation example and make sure to add read/write scope query parameters to GetToken request). RestClient classes with read/write authenticators can be made via singleton class.

Commit:
Develop client code to get bearer tokens with read and write scopes separately. Using RestSharp AuthenticatorBase develop custom authenticators that get token with “read” and “write” scope (check rest sharp documentation example and make sure to add read/write scope query parameters to GetToken request). RestClient classes with read/write authenticators can be made via singleton class.

Commit:
Small cleanup.

alexsavenok-savenduk commented on RestInfrastructure/Authenticators/ApiReadAuthenticator.cs:
```diff
@@ -0,0 +1,51 @@
+
+        public ApiReadAuthenticator() : base("")
+        {
+            _baseUrl = "http://localhost:49000";
```
I would store this data in some separate class or file to avoid duplicating

Kravchook replied:
I have added Settings folder to manage different settings. I am storing this data in appsettings.json file now. Please check this out.

alexsavenok-savenduk reviewed (approved):
LGTM!

Commit:
Updated solution's name.
Added Core project for basic solution's infrastructure and configuration settings.

Commit:
Added Settings folder to store settings.
